### PR TITLE
Added the certificate-acl service to the docker-compose file

### PR DESCRIPTION
**Pull request by rascaraficci**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)
The docker-compose file is missing the certificate-acl service

* **What is the new behavior (if this is a feature change)?**
Added the certificate-acl service to the docker-compose file

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
dojot/dojot#1939

* **Other information**:

### CODE REVIEW

**Review by mprevide — APPROVED** [review]

**Review by marianoleonardo — APPROVED** [review]
LGTM!

**Review by mprevide — APPROVED** [review]
I can't believe I didn't see the image

**Review by mprevide — CHANGES_REQUESTED** [review]
the img actually is dojot/dojot-acl

File: docker-compose.yml [hunk 1]
```diff
@@ -582,15 +582,41 @@ services:
...
+      options:
+        max-size: 20m
+        max-file: '5'
+
+  certificate-acl:
+    image: dojot/certificate-acl:development
+    depends_on:
+      - acl-redis
+      - kafka
+      - x509-ejbca
```
Comment (mprevide) [inline docker-compose.yml:601]: x509-identity-mgmt?

Comment (rascaraficci) [reply docker-compose.yml]: :disappointed:

File: docker-compose.yml [hunk 2]
```diff
@@ -582,15 +582,41 @@ services:
...
+    volumes:
+      - acl-redis-volume:/data
+    logging:
+      driver: json-file
+      options:
+        max-size: 20m
+        max-file: '5'
+
+  certificate-acl:
+    image: dojot/certificate-acl:development
```
Comment (mprevide) [inline docker-compose.yml:597]: the img actually is dojot/dojot-acl

Comment (mprevide) [reply docker-compose.yml:597]: maybe we can switch to the acl-certificate in github actions (yml)

Comment (mprevide) [reply docker-compose.yml:597]: I only realized why I put the acl in the version I'm working on, which is for 0.8

Comment (mprevide) [reply docker-compose.yml:597]: https://github.com/dojot/dojot/pull/2132

**Review by mprevide — APPROVED** [review]

**Review by marianoleonardo — APPROVED** [review]

**Pull request merged**